hippunfold/workflow/scripts/fillbadvertices.py

```diff
@@ -0,0 +1,59 @@
+import nibabel as nib
```

Comment:
i believe it was more than a rename, since it also tried to do outlier detection. I will test and then consider restoring the previous behaviour though, since this may not be needed